pkgs/development/python-modules/eiswarnung/default.nix

```
@@ -13,7 +13,7 @@
 
 buildPythonPackage rec {
   pname = "eiswarnung";
-  version = "1.2.0";
+  version = "2.0.0";
   format = "pyproject";
 
   disabled = pythonOlder "3.9";
```

```suggestion
  disabled = pythonOlder "3.11";
```
Python3.10 is no longer supported.
https://github.com/klaasnicolaas/python-eiswarnung/blob/ff6b274f695730c9e0f8f331b84d0823a104ac07/pyproject.toml#L29

---

Thanks